routers/web/repo/actions/view.go

```diff
@@ -572,3 +573,43 @@ func ArtifactsDownloadView(ctx *context_module.Context) {
 		}
 	}
 }
+
+func DisableWorkflowFile(ctx *context_module.Context) {
+	disableOrEnableWorkflowFile(false, ctx)
+}
+
+func EnableWorkflowFile(ctx *context_module.Context) {
+	disableOrEnableWorkflowFile(true, ctx)
+}
+
+func disableOrEnableWorkflowFile(isEnable bool, ctx *context_module.Context) {
```

Comment:
I think it's a good idea to keep the ctx as the first parameter.
```suggestion
func disableOrEnableWorkflowFile(ctx *context_module.Context, isEnable bool) {
```